src/software/ai/hl/stp/tactic/attacker/attacker_tactic_shoot_goal_test.cpp

```diff
@@ -60,7 +60,7 @@ INSTANTIATE_TEST_CASE_P(
     ShootGoalEnvironment, AttackerTacticShootGoalTest,
     ::testing::Values(
         // enemy goal blocked by enemy robots with enemy threat right
-        std::make_tuple(BallState(Point(2, 1), Vector()), Point(1, 1),
+        std::make_tuple(BallState(Point(2, 1), Vector()), Point(1, 1.05),
```

Comment:
why did this have to change?

Reply:
Because the robot in that test would fall in an infinite oscillation cycle between two positions which it should kick next from. Moving it slightly ended up being enough to make it not be in the infinite cycle.

Reply:
ok, please attach a replay and a commit to an issue so it can be fixed, otherwise lgtm

Reply:
Added #2693